packages/flutter/lib/src/material/tabs.dart

```diff
@@ -555,6 +555,11 @@ class _IndicatorPainter extends CustomPainter {
     final Rect fromRect = indicatorRect(size, from);
     final Rect toRect = indicatorRect(size, to);
     _currentRect = Rect.lerp(fromRect, toRect, (value - from).abs());
+
+    if (indicatorSize == TabBarIndicatorSize.label) {
```

Comment:
Can you change this to use a switch statement for assigning rect based on `TabBarIndicatorSize`?

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values

Reply:
On 67599575fe733c8a844bab845cd463c9d8a30d9d I've changed it to a switch, and thanks to that I noticed that the `indicatorSize` property was declared as nullable, but null is never provided. I took the chance to clean it up, since it is a private class.